lib/commands/clipboard.js

```diff
@@ -0,0 +1,40 @@
+import { exec } from 'teen_process';
+
+const commands = {};
+
+/**
+ * Sets the clipboard's content on the Windows PC.
+ *
+ * @param {!string} content - The content to be set as base64 encoded string.
+ * @param {?string} contentType [plaintext] - The type of the content to set.
+ *                                            Only `plaintext` and 'image' are supported.
+ */
+commands.setClipboard = async function setClipboard (content, contentType) {
+  if (contentType === 'plaintext') {
+    return await exec('powershell', ['-command', `Set-Clipboard -Value '${contentType}'`]);
+  } else if (contentType === 'image') {
+    return await exec('powershell', ['-command', `$img=[Drawing.Bitmap]::FromStream([IO.MemoryStream][Convert]::FromBase64String(${content}));[System.Windows.Forms.Clipboard]::SetImage($img);$img.Dispose();`]);
```

Comment:
this command is not supposed to return anything

Reply:
Duh me - lazy copy and paste after doing gets first